src/streams/StreamScreen.js

```diff
@@ -59,29 +60,31 @@ class StreamScreen extends PureComponent<Props> {
     const { styles } = this.context;
 
     return (
-      <Screen title="Stream" padding>
-        <StreamCard stream={stream} subscription={subscription} />
-        <OptionDivider />
+      <Screen title="Stream">
+        <View style={styles.screenPadding}>
+          <StreamCard stream={stream} subscription={subscription} />
+        </View>
         <OptionRow
           label="Pinned"
           defaultValue={subscription.pin_to_top}
           onValueChange={this.handleTogglePinStream}
         />
-        <OptionDivider />
         <OptionRow
           label="Muted"
           defaultValue={stream.in_home_view === false}
           onValueChange={this.handleToggleMuteStream}
         />
-        <OptionDivider />
         <OptionRow
           label="Notifications"
           defaultValue={subscription.push_notifications}
           onValueChange={this.toggleStreamPushNotification}
           customStyle={this.context.styles.backgroundColor}
         />
-        <ZulipButton style={styles.marginTop} text="Topics" onPress={this.handleTopics} />
-        <ZulipButton style={styles.marginTop} text="Edit" onPress={this.handleEdit} />
+        <OptionDivider />
```

Comment:
The idea of changing the style of `OptionDivider` was that it would be used to divide groups of settings, so using it to divide the options and the buttons seemed correct to me.

But if you are absolutely sure it is not required there, I will go ahead and remove it.

Reply:
Separately from that, imagine we wanted to add the border or any other effect.
That logic would be better extracted as a component.

`OptionGroup` can be just: 
```js
<View style={styles.optionGroup}>
```
And then you reuse it everywhere else that is needed.